contrib/vagrant/provision-full.sh

```diff
@@ -3,8 +3,9 @@ set -euo pipefail
 IFS=$'\n\t'
 USERNAME="${1:-vagrant}"
 
-yum update -y
-yum install -y docker-io git vim golang e2fsprogs tmux httpie ctags hg bind-utils which
+#yum update for fedora provider in vagrant breaks vboxfs 
+#yum update -y
+yum install -y docker-io git vim golang e2fsprogs tmux httpie ctags hg bind-utils which ethtool
```

Comment:
`ethtool` (and probably all others?) should be part of the base image, @kargakis [added it there](https://github.com/openshift/vagrant-openshift/pull/332) a few weeks ago, we just need to have the base image updated.